src/components/LayerUI.tsx

```diff
@@ -137,7 +137,7 @@ export const LayerUI = React.memo(
               </Section>
               {showSelectedShapeActions(appState, elements) && (
                 <Section
-                  className="App-right-menu"
```

Comment:
`App-right-menu` is it's referenced anywhere else?

Reply:
It's not being used. I'll remove it.